python/cm/issue.py

```diff
@@ -114,54 +153,41 @@ def get_issue(obj):   # pylint: disable=too-many-branches
 
 
 def cook_issue(obj, name='name', name_obj=None, iss=None):
-    if not name_obj:
-        name_obj = obj
-    if not iss:
-        if obj:
-            iss = obj.issue
-        else:
-            iss = {}
-    if iss:
-        return {
-            'id': obj.id,
-            'name': getattr(name_obj, name),
-            'issue': iss,
-        }
-    return None
+    result = {
+        'id': getattr(obj, 'id'),
+        'name': getattr(name_obj or obj, name),
+        'issue': iss or getattr(obj, 'issue', {})
+    }
+    return result if all(result.values()) else {}
 
 
 def check_cluster_issue(cluster):
-    issue = {}
-    if not check_config(cluster):
-        issue['config'] = False
-    if not check_required_services(cluster):
-        issue['required_service'] = False
-    if not check_required_import(cluster):
-        issue['required_import'] = False
-    if not check_hc(cluster):
-        issue['host_component'] = False
-    return issue
+    return {
```

Comment:
It is the change of semantic. Now frontend assumed to get empty issue, if issue is absent. I think we should keep this behavior.

Reply:
Nope, same logic remains, just moved in single place to line 119 of this file